### DISCUSSION TITLE
Certificate revoked, can't access from Chrome or Safari

### DISCUSSION THREAD

**Discussion by abrichr** [fastapi/full-stack-fastapi-template#1809] [Questions]
Issue: https://www.bleepingcomputer.com/news/security/lets-encrypt-is-revoking-lots-of-ssl-certificates-in-two-days/

Resolution:
```
# Delete traefik stack
docker stack rm traefik

# Delete traefik volume
docker volume ls (should see something like traefik_traefik-public-certificates)
docker volume rm traefik_traefik-public-certificates

# Populate env variables for deploying traefik (reference: https://github.com/tiangolo/full-stack-fastapi-postgresql/issues/322)
source ~/.profile
export EMAIL=$TRAEFIK_EMAIL
export HASHED_PASSWORD=$(openssl passwd -apr1 $TRAEFIK_PASSWORD)

# Re-deploy traefik stack
docker stack deploy -c traefik.yml traefik
```

Thanks to @0xstnick!

**Comment by lacriment** [comment]
I followed your steps and I get TRAEFIK DEFAULT CERT instead of Let's Encrypt certificate. on stag.domain.com but I have correct lets encrypt cert on traefik.stag.domain.com. What am I missing?
<img width="637" alt="Screen Shot 2022-02-01 at 08 46 59" src="https://user-images.githubusercontent.com/6463229/151918925-4b15a1b3-4f4b-4e80-9558-d94cc2690938.png">

**Comment by jashwant** [comment]
@lacriment, I followed [this traefik article](https://traefik.io/blog/how-to-force-update-lets-encrypt-certificates/) and it worked for me.
